starter_code/src/movies.js

```diff
@@ -1,20 +1,132 @@
 /* eslint no-restricted-globals: 'off' */
-// Turn duration of the movies from hours to minutes 
+// Turn duration of the movies from hours to minutes
+//dont do this one!!!!!!!!!
 
+// function turnHoursToMinutes(movies) {
+//   var durations = movies.forEach(val => {
+//     return movies.duration;
+//   });
 
-// Get the average of all rates with 2 decimals 
+//   var num = durations.split(" ");
 
+// "2h", "22min"
+// "1h","30min"
 
-// Get the average of Drama Movies
+//var minutes = movies.hour() * 60 + movies.minute();
+// }
 
+// ////////Get the average of all rates with 2 decimals
 
-// Order by time duration, in growing order
+function ratesAverage(movies) {
+  var total = 0;
+  movies.forEach(val => {
+    total += Number(val.rate);
+  });
+  return total / movies.length;
+}
 
+// //////////Get the average of Drama Movies
 
-// How many movies did STEVEN SPIELBERG
+function dramaMoviesRate(movies) {
+  var dtotal = 0;
 
+  var dramaMovies = movies.filter(movie => {
+    return movie.genre.includes("Drama");
+  });
 
-// Order by title and print the first 20 titles
+  // console.log("all drama movies are : " + dramaMovies);
 
+  dramaMovies.forEach(dramaMovie => {
+    dtotal += Number(dramaMovie.rate);
+  });
+  // console.log("total of all drama movies: " + dtotal);
 
-// Best yearly rate average
+  return dtotal / dramaMovies.length;
```

Comment:
Check .toFixed() method when you are asked to return "n" number after the dot of a float number